chia/full_node/full_node.py

```diff
@@ -97,54 +97,55 @@ class FullNode:
     sync_store: Any
     coin_store: CoinStore
     mempool_manager: MempoolManager
-    _sync_task: Optional[asyncio.Task]
-    _init_weight_proof: Optional[asyncio.Task] = None
+    _sync_task: Optional[asyncio.Task[None]]
+    _init_weight_proof: Optional[asyncio.Task[None]] = None
     blockchain: Blockchain
-    config: Dict
+    config: Dict[str, Any]
     server: Any
     log: logging.Logger
     constants: ConsensusConstants
     _shut_down: bool
     root_path: Path
-    state_changed_callback: Optional[Callable]
+    state_changed_callback: Optional[Callable[..., Any]]
     timelord_lock: asyncio.Lock
     initialized: bool
     multiprocessing_start_context: Optional[BaseContext]
     weight_proof_handler: Optional[WeightProofHandler]
-    _ui_tasks: Set[asyncio.Task]
+    _ui_tasks: Set[asyncio.Task[None]]
     _blockchain_lock_queue: LockQueue
     _blockchain_lock_ultra_priority: LockClient
     _blockchain_lock_high_priority: LockClient
     _blockchain_lock_low_priority: LockClient
-    _transaction_queue_task: Optional[asyncio.Task]
-    simulator_transaction_callback: Optional[Callable]
+    _transaction_queue_task: Optional[asyncio.Task[None]]
+    simulator_transaction_callback: Optional[Callable[..., Any]]
 
     def __init__(
         self,
-        config: Dict,
+        config: Dict[str, Any],
         root_path: Path,
         consensus_constants: ConsensusConstants,
-        name: str = None,
-    ):
+        name: str = "",
```

Comment:
This, and drop the conditional below?

```suggestion
        name: str = __name__,
```